tests/kernel/iop/interrupt/register.expected

```diff
@@ -0,0 +1,10 @@
+-- TEST BEGIN
+multiple register:
+  register when no handler is present: 0
+  register when handler is already present: -104
+  handler result: 43690
```

Comment:
It's the handler's parameter that comes from here: https://github.com/unknownbrackets/ps2autotests/pull/22/files#diff-43f07e7b7c8b707fd61bdeeb5c40f43bR34. It's consistent between runs.